index.js

```diff
@@ -34,7 +34,8 @@ function createLoaderPipeline (options, assets) {
 
                 return { 
                     code: transpiled.css.toString(),
-                    map: transpiled.map.toString()
+                    map: transpiled.map.toString(),
+                    watchFiles: transpiled.stats.includedFiles || []
```

Comment:
This should be noted in the README.md file as an optional return parameter for loaders.

Reply:
Right, I'll add it and make it optional, since custom loaders will crash if not defining the watchFile property this way.